src/celeritas/phys/PhysicsParams.hh

```diff
@@ -71,42 +134,36 @@ class ParticleParams;
  *   \c spline_eloss_order: the order of interpolation to be used for the
  *   spline interpolation. If it is 1, then the existing linear interpolation
  *   is used. If it is 2+, the spline interpolation is used for energy loss
- *   using the specified order. Default value is 1
- *
- * NOTE: min_range/max_step_over_range are not accessible through Geant4, and
- * they can also be set to be different for electrons, mu/hadrons, and ions
- * (they are set in Geant4 with \c G4EmParameters::SetStepFunction()).
+ *   using the specified order. Default value is 1.
  */
 struct PhysicsParamsOptions
 {
-    using Energy = units::MevEnergy;
-
     //!@{
     //! \name Range calculation
-    real_type min_range = real_type{1} * units::millimeter;
-    real_type max_step_over_range = 0.2;
-    real_type fixed_step_limiter = 0;
+    real_type fixed_step_limiter{0};
     //!@}
 
     //!@{
     //! \name Energy loss
-    real_type min_eprime_over_e = 0.8;
-    real_type linear_loss_limit = 0.01;
-    Energy lowest_electron_energy = Energy{0.001};
+    real_type min_eprime_over_e{0.8};
+    real_type linear_loss_limit{0.01};
     //!@}
 
     //!@{
     //! \name Multiple scattering
-    real_type lambda_limit = real_type{1} * units::millimeter;
-    real_type range_factor = 0.04;
-    real_type safety_factor = 0.6;
-    MscStepLimitAlgorithm step_limit_algorithm{MscStepLimitAlgorithm::safety};
+    real_type lambda_limit{real_type{1} * units::millimeter};
+    real_type safety_factor{0.6};
     //!@}
 
-    real_type secondary_stack_factor = 3;
-    bool disable_integral_xs = false;
+    //!@{
+    //! \name Particle-dependent parameters
+    ParticleOptions em{ParticleOptions::default_em()};
+    ParticleOptions muhad{ParticleOptions::default_muhad()};
```

Comment:
Not trying to rock the boat, but since we now have `is_heavy` (which I like, since that's really what characterizes the different slowing down behaviors) should we make these light/heavy rather than em/muhad? I'm fine with keeping them as is to avoid you spending your whole life renaming things 😅